internal/cfgplugins/interface.go

```diff
@@ -37,15 +40,469 @@ const (
 )
 
 var (
-	opmode uint16
-	once   sync.Once
+	opmode   uint16
+	once     sync.Once
+	lBandPNs = map[string]bool{
+		"DP04QSDD-LLH-240": true, // Cisco QSFPDD Acacia 400G ZRP L-Band
+		"DP04QSDD-LLH-00A": true, // Cisco QSFPDD Acacia 400G ZRP L-Band
+		"DP08SFP8-LRB-240": true, // Cisco OSFP Acacia 800G ZRP L-Band
+		"C-OS08LEXNC-GG":   true, // Nokia OSFP 800G ZRP L-Band
+		"176-6490-9G1":     true, // Ciena OSFP 800G ZRP L-Band
+	}
 )
 
 // Temporary code for assigning opmode 1 maintained until opmode is Initialized in all .go file
 func init() {
 	opmode = 1
 }
 
+// OperationalModeList is a type for a list of operational modes in uint16 format.
+type OperationalModeList []uint16
+
+// String returns the string representation of the list of operational modes.
+func (om *OperationalModeList) String() string {
+	var s []string
+	for _, v := range *om {
+		s = append(s, fmt.Sprintf("%v", v))
+	}
+	return strings.Join(s, ",")
+}
+
+// Set sets the list of operational modes from the string representation.
+func (om *OperationalModeList) Set(value string) error {
```

Comment:
Here and throughout, modify exported function names of the to have content that makes these cfgplugins functions indicate they are for interface pmd.  
A suggested suffix could be `InterfacePMDOperationalMode`.

```suggestion
func (om *OperationalModeList) SetInterfacePMDOperationalMode(value string) error {
```